src/controllers/controllerengine.cpp

```diff
@@ -1152,10 +1152,10 @@ void ControllerEngine::scratchProcess(int timerId) {
     // If we're ramping to end scratching and the wheel hasn't been turned very
     // recently (spinback after lift-off,) feed fixed data
     if (m_ramp[deck] &&
-        ((Time::elapsedMsecs() - m_lastMovement[deck]) > 0)) {
+        ((Time::elapsed() - m_lastMovement[deck]) > mixxx::Duration::fromSeconds(0))) {
```

Comment:
This is now always true due to the higher precision. 
The old solution was also not that reliable. Maybe a source of the wobbling sound if you try to controller scratch with rate = 1.0.